src/support/filename.c

```diff
@@ -48,3 +48,59 @@ __wt_nfilename(
 
 	return (0);
 }
+
+/*
+ * __wt_remove_if_exists --
+ *	Remove a file if it exists.
+ */
+int
+__wt_remove_if_exists(WT_SESSION_IMPL *session, const char *name)
+{
+	int exist;
+
+	WT_RET(__wt_exist(session, name, &exist));
+	if (exist)
+		WT_RET(__wt_remove(session, name));
+	return (0);
+}
+
+/*
+ * __wt_fp_open --
+ *	Open a FILE handle.
+ */
+int
+__wt_fp_open(WT_SESSION_IMPL *session, const char *name, FILE **fpp)
+{
+	WT_DECL_RET;
+	char *path;
+
+	WT_RET(__wt_filename(session, name, &path));
+
+	if ((*fpp = fopen(path, "w")) == NULL)
```

Comment:
This needs "b" on Windows so that it does not do CR & LF translation. https://msdn.microsoft.com/en-us/library/yeby3zcb.aspx?f=255&MSPPError=-2147217396